py/selenium/webdriver/common/timeouts.py

```diff
@@ -39,9 +34,7 @@ class JSONTimeouts(TypedDict, total=False):
 class _TimeoutsDescriptor:
     """Get or set the value of the attributes listed below.
 
-    _implicit_wait
-    _page_load
-    _script
+    _implicit_wait _page_load _script
```

Comment:
On a single line I think it becomes harder to read

Reply:
I tried to revert it but again the linter complained. I am not sure how it worked before.